DocumentDataAPI/DocumentDataAPI/appsettings.json

```diff
@@ -7,12 +7,12 @@
   },
   "AllowedHosts": "*",
   "Database": {
-    "Host": "localhost",
-    "Port": 5432,
-    "Database": "[INSERT DATABASE]",
-    "Schema": "document_data",
-    "Username": "[INSERT USERNAME]",
-    "Password": "[INSERT PASSWORD]"
+    "Host": "#{DATABASE_HOST}#",
+    "Port": "#{DATABASE_PORT}#",
+    "Database": "#{DATABASE_DATABASE}#",
+    "Schema": "#{DATABASE_SCHEMA}#",
+    "Username": "#{DATABASE_USERNAME}#",
+    "Password": "#{DATABASE_PASSWORD}#"
```

Comment:
Påvirker det her, om projektet kan køres lokalt, eller er den okay med denne konfiguration?

Reply:
`appsettings.json` er "nederst" i settings-hierarkiet, så de vil f.eks. blive overskrevet af environment variables, User Secrets, eller `appsettings.local.json`. Så det vil ikke ændre på noget ift. at køre det lokalt. (Men det kræver så at vi ikke ændrer på den her fil, for at felterne kan blive korrekt erstattet)